centaur/src/main/resources/standardTestCases/bad_stderr_input.test

```diff
@@ -0,0 +1,7 @@
+# Verifies that if we use stderr in a workflow level, the workflow fails properly.
```

Comment:
You can now remove this and other centaur tests as you have replaced it with unit tests.